### PR TITLE
fix cdc redo log storage configuration

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
ticdc support use gcs/azure as external storage in redo log.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions) Merge only after release of 6.5.1
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions) Merge only after release of 6.1.4
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-sink-to-mysql.md [hunk 1]
```diff
@@ -90,9 +90,13 @@ MTIzNDU2
 - 执行耗时很长的 DDL 语句，例如：add index
 - 使用过于激进的 PD 调度策略，导致频繁 region leader 迁移或 region merge/split
 
+> **注意：**
+>
+> TiCDC 最终一致性功能从 v6.1.1 开始支持兼容 Amazon S3 协议的对象存储，从 v6.1.4、v6.5.1 开始支持兼容 GCS/Azure 协议的对象存储。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-mysql.md:95]:
```suggestion
> TiCDC 最终一致性功能从 v6.1.1 开始支持兼容 Amazon S3 协议的对象存储，从 v6.1.4 开始支持兼容 GCS/Azure 协议的对象存储。
```
TiCDC 6.5.1 还没有发布

File: ticdc/ticdc-sink-to-mysql.md [hunk 2]
```diff
@@ -78,9 +78,9 @@ MTIzNDU2
 
 ## 灾难场景的最终一致性复制
 
-从 v6.1.1 版本开始容灾场景下的最终一致性复制功能 GA。TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的 S3 存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
+从 v6.1.1 版本开始容灾场景下的最终一致性复制功能 GA。TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的对象存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:81]:
```suggestion
从 v6.1.1 版本开始，容灾场景下的最终一致性复制功能 GA。TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的对象存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 3]
```diff
@@ -78,9 +78,9 @@ MTIzNDU2
...
 
-目前，TiCDC 支持将 TiDB 集群的增量数据复制到 TiDB 或兼容 MySQL 的数据库系统（包括 Aurora、MySQL 和 MariaDB）。当上游发生灾难时，如果 TiCDC 正常运行且上游 TiDB 集群没有出现数据复制延迟大幅度增加的情况，下游集群可以在 5 分钟之内恢复集群，并且最多丢失出现问题前 10 秒钟的数据，即 RTO <= 5 mins, P95 RPO <= 10s。
+目前，TiCDC 支持将 TiDB 集群的增量数据复制到 TiDB 或兼容 MySQL 的数据库系统（包括 Aurora、MySQL 和 MariaDB）。当上游发生灾难时，如果出现故障前 TiCDC 正常运行且上游 TiDB 集群没有出现数据复制延迟大幅度增加的情况，下游集群可以在 5 分钟之内恢复集群，并且最多丢失出现问题前 10 秒钟的数据，即 RTO <= 5 mins, P95 RPO <= 10s。
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:83]:
```suggestion
目前，TiCDC 支持将 TiDB 集群的增量数据复制到 TiDB 或兼容 MySQL 的数据库系统（包括 Aurora、MySQL 和 MariaDB）。如果 TiCDC 在上游发生灾难前正常运行，且上游 TiDB 集群没有出现数据复制延迟大幅度增加的情况，灾难发生后，下游集群可以在 5 分钟之内恢复集群，并且最多丢失出现问题前 10 秒钟的数据，即 RTO <= 5 min，P95 RPO <= 10s。
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 4]
```diff
@@ -90,9 +90,13 @@ MTIzNDU2
...
 - 使用过于激进的 PD 调度策略，导致频繁 region leader 迁移或 region merge/split
 
+> **注意：**
+>
+> TiCDC 最终一致性功能从 v6.1.1 开始支持兼容 Amazon S3 协议的对象存储，从 v6.1.4、v6.5.1 开始支持兼容 GCS/Azure 协议的对象存储。
+
 ### 使用前提
 
-- 准备好具有高可用的 S3 存储或 NFS 系统，用于存储 TiCDC 的实时增量数据备份文件，在上游发生灾难情况下该文件存储可以访问。
+- 准备好具有高可用的对象存储或 NFS 系统，用于存储 TiCDC 的实时增量数据备份文件，在上游发生灾难情况下该文件存储可以访问。
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:99]:
```suggestion
- 准备高可用的对象存储或 NFS 系统，用于存储 TiCDC 的实时增量数据备份文件，在上游发生灾难情况下，该文件存储可以访问。
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 5]
```diff
@@ -108,8 +112,8 @@ max-log-size = 64
 # 刷新或上传 redo log 至 S3 的间隔，单位毫秒，建议该参数 >= 2000。
 flush-interval = 2000
 
-# 存储 redo log 的形式，包括 nfs（NFS 目录），S3（上传至S3）
-storage = "s3://logbucket/test-changefeed?endpoint=http://$S3_ENDPOINT/"
+# redo log 备份文件的地址，支持的 scheme 包括 nfs（NFS 目录）和 S3/GCS/Azure（上传至对象存储）
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:115]:
```suggestion
# redo log 备份文件的地址，支持的 scheme 包括 nfs（NFS 目录）和 Amazon S3、GCS 和 Azure（上传至对象存储）。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: ticdc/ticdc-sink-to-mysql.md [hunk 6]
```diff
@@ -78,9 +78,9 @@ MTIzNDU2
 
 ## 灾难场景的最终一致性复制
 
-从 v6.1.1 版本开始容灾场景下的最终一致性复制功能 GA。TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的 S3 存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
+从 v6.1.1 版本开始，容灾场景下的最终一致性复制功能 GA。TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的对象存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-mysql.md:81]:
```suggestion
从 v6.1.1 版本开始，容灾场景下的最终一致性复制功能 GA。从 v5.3.0 开始，TiCDC 支持将上游 TiDB 的增量数据备份到下游集群的对象存储或 NFS 文件系统。当上游集群出现了灾难，完全无法使用时，TiCDC 可以将下游集群恢复到最近的一致状态，即提供灾备场景的最终一致性复制能力，确保应用可以快速切换到下游集群，避免数据库长时间不可用，提高业务连续性。
```

**Pull request merged**